### DISCUSSION TITLE
selecting line start and end also try to copy text

### DISCUSSION THREAD

**Discussion by msedgren** [talonhub/community#1630] [Q&A]
There is a problem in IntelliJ where it does funky stuff when used with the commands 'copy way right'... I think I traced this down to the functions select_line_start and select_line_end in core/edit/edit.py. These functions seem to be attempting to copying the current selected text and moving if there is something selected. I am a bit confused on why this is happening as it seems to be a strange side effect for the functions mentioned above. I'm wondering if these functions should be changed or if something in IntelliJ files should be altered to resolve this. Any help or comments are greatly appreciated.

**Comment by alexander-clarke** [comment]
Hey, I've also been having this issue. It also causes issues in other editors as well I think
My suggested fix is that we no longer use the functions select_line_start/end
I implemented this in #1636 but it is not yet merged. Also not sure if there is a preferred different solution
